CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOSUITests/Info.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same as above

Reply:
These are autogenerated for the app. Its ok to leave them there. There are no tests in the file. I don't want to break the build.